terraform/environments/data-platform/tests/run_smoke_tests.sh

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
This script looks really useful 👍🏻

Reply:
it doesn't test registration is the only thing, that can be part of another ticket

Reply:
Yeah agreed. Should be easy to add once the workflow is sorted.